src/views/EditorHome.vue

```diff
@@ -114,6 +114,7 @@
                           text-color="display-on-primary"
                           :disable="uiLocked"
                           @click="addAudioItem"
+                          aria-label="行を追加"
```

Comment:
使い方の用語に合わせると「テキストの追加」のが合ってるかもです！
```suggestion
                          aria-label="テキストを追加"
```